cypress/integration/console/applications/api-keys/api-keys.spec.js

```diff
@@ -0,0 +1,100 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+describe('Application API keys', () => {
+  const apiKeyName = 'api-test-key'
+  const applicationId = 'api-keys-test-app'
+  const application = { ids: { application_id: applicationId } }
+  const userId = 'main-api-key-test-user'
+  const user = {
+    ids: { user_id: userId },
+    primary_email_address: 'create-api-key-test-user@example.com',
+    password: 'ABCDefg123!',
+    password_confirm: 'ABCDefg123!',
+  }
+
+  before(() => {
+    cy.dropAndSeedDatabase()
+
+    cy.createUser(user)
+    cy.loginConsole({
+      user_id: userId,
+      password: user.password,
+    })
+    cy.createApplication(application, userId)
+    cy.clearLocalStorage()
+    cy.clearCookies()
+  })
+
+  beforeEach(() => {
+    cy.loginConsole({ user_id: user.ids.user_id, password: user.password })
+  })
+
+  it('succeed adding new api key', () => {
```

Comment:
Please read the docs section related to writing tests https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.10/DEVELOPMENT.md#L770-L894